tests/Neo.UnitTests/SmartContract/Native/UT_RoleManagement.cs

```diff
@@ -90,6 +91,61 @@ public void TestSetAndGet()
             (ret as VM.Types.Array).Count.Should().Be(0);
         }
 
+        [TestMethod]
+        public void TestDesignateP2PNotary()
```

Comment:
Maybe this can be generalized with `TestSetAndGet` for all roles?

Reply:
Sure, fixed in a separate commit.